unit_tests/filter_test.cc

```diff
@@ -1,15 +1,19 @@
 #include "../include/filter/filter.hh"
 #include <chrono>
-#include <cmath>
 #include <gtest/gtest.h>
 #include <vector>
 
+// This is a unit test, magic numbers are called test data here
+// NOLINTBEGIN(cppcoreguidelines-avoid-magic-numbers)
```

Comment:
are unit tests now static analysed?

Reply:
nope, but the SA stuff still runs in my linter and if we do start running SA on our tests then this will be useful

Reply:
Ah, makes sense